demo.html

```diff
@@ -48,7 +48,8 @@ <h3>and small.</h3>
     <h3>And unordered</h3>
     <div id="cluster3"></div>
     <script src="lib/jquery.js"></script>
-    <script src="lib/d3.v2.js"></script>
+    <script src="https://unpkg.com/d3"></script>
+    <!--<script src="lib/d3.v2.js"></script>  d3v2-->
```

Comment:
Please don't leave dead code around. We use a version control system so we can go back to older versions.